setup.py

```diff
@@ -5,14 +5,18 @@
     setup
 )
 
+with open(os.path.join('requirements.txt'), 'r') as f:
+    REQUIRED_PACKAGES = f.readlines()
 
-packages = [x for x in find_packages()
+PACKAGES = find_packages()
```

Comment:
I would inline that as it could be confusing to have two "packages" variables. You could capitilise `packages` below if you prefer.